cmd/soroban-cli/src/config/sign_with.rs

```diff
@@ -38,14 +47,16 @@ pub struct Args {
         env = "STELLAR_SIGN_WITH_SECRET"
     )]
     pub sign_with_key: Option<String>,
-    /// Sign with labratory
+    /// Sign with laboratory
+    #[arg(long, conflicts_with = "sign_with_key", env = "STELLAR_SIGN_WITH_LAB")]
+    pub sign_with_lab: bool,
+    /// Lab URL for `sign_with_lab`
     #[arg(
         long,
-        conflicts_with = "sign_with_key",
-        env = "STELLAR_SIGN_WITH_LABRATORY",
-        hide = true
+        env = "STELLAR_SIGN_WITH_LAB_URL",
+        default_value = "https://lab.stellar.org/transaction/cli-sign"
     )]
-    pub sign_with_lab: bool,
+    pub lab_url: String,
```

Comment:
Should it also be `conflicts_with = "sign_with_key"`?

Reply:
Good call! I'll push up a change shortly